docs/pages/branding/careers.tsx

```diff
@@ -0,0 +1,491 @@
+            />
+            <Divider sx={{ my: { xs: 1, sm: 2 }, borderColor: 'primaryDark.600' }} />
+            <Typography
+              component="h3"
+              variant="h5"
+              color="primary"
+              fontWeight="extraBold"
+              sx={{ mb: 2 }}
+            >
+              Customer Success
+            </Typography>
+            <Role
+              title="Support Engineer"
+              description="Ensure that our users wildly succeed on their journey with Material-UI. You’ll directly work with users, customers, and potential customers to unblock them from using the products, triage and resolve issues, and use this direct feedback to drive direct improvements in Material-UI."
+            />
+          </Container>
+        </Box>
+      </MuiThemeProvider>
+      {/* Frequently asked questions */}
+      <Box>
+        <Container sx={{ py: { xs: 4, md: 8 } }}>
+          <Typography variant="h2" sx={{ my: 1 }}>
+            Frequently Asked Questions
+          </Typography>
+          <Grid container alignItems="start" spacing={3} sx={{ my: 1 }}>
+            <Grid item xs={12} md={6}>
+              <Question question="Are there application deadlines?" defaultShowAnswer>
+                <Typography variant="body2" color="text.secondary" sx={{ mt: 1 }}>
+                  No. If a job is visible on our careers page, then you can still apply.
+                </Typography>
+              </Question>
+            </Grid>
+            <Grid item xs={12} md={6}>
+              <Question question="Does MUI do whiteboarding during interviews?">
+                <Typography variant="body2" color="text.secondary" sx={{ mt: 1 }}>
+                  ...{/* Need Answer */}
+                </Typography>
+              </Question>
+            </Grid>
+            <Grid item xs={12} md={6}>
+              <Question question="Does MUI offer contract job opportunities?">
+                <Typography variant="body2" color="text.secondary" sx={{ mt: 1 }}>
+                  ...{/* Need Answer */}
+                </Typography>
+              </Question>
+            </Grid>
+            <Grid item xs={12} md={6}>
+              <Widget
+                title="Got any questions unanswered or need more help?"
+                url={'mailto:contact@material-ui.com'}
+                linkText="Contact us"
+              >
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 1 }}>
+                  We&apos;re to help you with any other question you have about our hiring process.
+                </Typography>
```

Comment:
For the FAQ section, can you use the same code as `DesignKitFAQ.tsx` (it uses accordion)?